react/__tests__/index.test.tsx

```diff
@@ -712,4 +712,23 @@ describe('GA4 events', () => {
       })
     })
   })
+  describe('sign_up', () => {
```

Comment:
Nitpick 💅: Just an empty line to separate them better.
```suggestion

  describe('sign_up', () => {
```